### PR TITLE
Add IRSAP Now Integration

### PR DESCRIPTION
**Pull request by Sim0cYz**
## Proposed change

Icons and logo for the IRSAP Now heaters custom integration.

## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [x] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 
- Link to custom integration repository: [https://github.com/hexCut/irsap-ha](https://github.com/hexCut/irsap-ha)

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
The domain of the integration in the linked repository doesn't match the domain used in this PR and doesn't match the integration folder name upstream.

CleanShot 2024-09-30 at 08 19 53@2x

Please ensure the domains match.

../Frenck

**Comment by Sim0cYz** [comment]
Hi @frenck , I talked with the other guys and we decided to keep `irsap-ha` as domain.
It will match in the custom component repository in the few hours. Thank you!

**Comment by valerix85** [comment]
Hi @frenck here is one of the developers writing, we have released in the “main” branch the very first version so we are affixed with what is required.
If anything else is needed we are here to help.

Thanks for your time

File: .vscode/settings.json
Comment (frenck) [inline .vscode/settings.json:1]: This is an unrelated change. Please revert this.

File: custom_integrations/irsap-ha/dark_icon.png
Comment (frenck) [inline custom_integrations/irsap-ha/dark_icon.png:1]: `irsap-ha` is not a valid integration domain, as the `-` cannot be used.

**Review by frenck — CHANGES_REQUESTED** [review]

File: custom_integrations/irsap-ha/dark_icon.png
Comment (valerix85) [reply custom_integrations/irsap-ha/dark_icon.png:1]: Can we use irsap_ha?

Comment (frenck) [reply custom_integrations/irsap-ha/dark_icon.png:1]: Yes, see: <https://developers.home-assistant.io/docs/creating_integration_manifest/#domain>

**Comment by valerix85** [comment]
I will do it immediately, can you approve it so?

**Comment by Sim0cYz** [comment]
meanwhile, I changed the folder name (now it's irsap_ha)

File: custom_integrations/irsap-ha/dark_icon.png
Comment (valerix85) [reply custom_integrations/irsap-ha/dark_icon.png:1]: I will do it immediately, can you approve it so?

File: .vscode/settings.json
Comment (valerix85) [reply .vscode/settings.json:1]: Hi @frenck, sorry for the insistence but I wanted to know if you had had time to look and let me know if there is anything else to fix on our side or if everything is ready for the release of the icon. So even with HACS we are ready to be published =)

Thank you

Comment (frenck) [reply .vscode/settings.json:1]: @valerix85 This is not waiting on me. This PR is in draft, meaning: It is not ready for review. The author can mark it ready for review when they have processed all comments (including this one, as this unrelated change is still in the PR).

Comment (valerix85) [reply .vscode/settings.json:1]: Sorry @frenck it was a misunderstanding with @Sim0cYz that now has resolved. I think we’re good to go 😊

Comment (frenck) [reply .vscode/settings.json:1]: We are not, this comment hasn't been resolved.

**Review by frenck — CHANGES_REQUESTED** [review]
Please resolve the open comment.

../Frenck

File: .vscode/settings.json
Comment (valerix85) [reply .vscode/settings.json:1]: Ok, I will guide @Sim0cYz  to resolve it. Many thanks and sorry to bother you again 😊

**Review by techunleadit — APPROVED** [review]

**Review by techunleadit — APPROVED** [review]

**Comment by techunleadit** [comment]
> Please resolve the open comment.
> 
> ../Frenck

done!

**Review by valerix85 — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (frenck) [inline .vscode/settings.json:1]: Alright, this file needs to be removed.

I've requested multiple times now to resolve this comment and it isn't happening.

I'm going to close this PR for now for that reason.

../Frenck

Comment (valerix85) [reply .vscode/settings.json:1]: Hi @frenck ,
I understand the frustration there is in these cases but @Sim0cYz not being practical didn't understand, I got added to his fork and fixed it.
Please bear with me and reopen so we can get the logo.
If there is anything else, please write to me so I don't prolong it more than necessary and don't take more of your time.

Thank you again for your help
